package.json

```diff
@@ -36,6 +36,7 @@
   "devDependencies": {
     "babel-eslint": "^10.1.0",
     "babel-jest": "^26.6.3",
+    "epubjs": "^0.3.88",
```

Comment:
The reason that it is not working, is probably because you need to import `epubjs` as well.
maybe in `index.js` or somewhere else like `import 'epubjs;` or `import {something}  from 'epubjs'`.
study how the `epubjs` library works and is imported.